can-view-parser.js

```diff
@@ -435,7 +445,7 @@ HTMLParser.parseAttrs = function(rest, handler){
 };
 
 HTMLParser.searchStartTag = function (html) {
-	var closingIndex = html.indexOf('>');
+	var closingIndex = HTMLParser.sanitizeAttrs(html).indexOf('>');
```

Comment:
When `searchStartTag` finds a tag, it's not called again until there's a need to find another tag.

e.g. `'<div foo=">"><hr foo=">" bar=\'>>->\' /></div>'` results in `searchStartTag` being called twice.
- first call: `<div foo=">"><hr foo=">" bar='>>->' /></div>`
- second call: `<hr foo=">" bar='>>->' /></div>`